### ISSUE TITLE
bug: can't override toolbar title mode

### ISSUE THREAD

**Issue by brandyscarney** [ionic-team/ionic#17876]
# Bug Report

**Ionic version:**

[x] **4.x**

**Current behavior:**
Set the mode on the `<ion-title>`, see that it uses the current mode still.

**Expected behavior:**
Use the mode provided.

**Steps to reproduce:**
https://codepen.io/brandyscarney/pen/bZZXrm

**Related code:**
I believe this is the problem in `ion-title`, but I'm not sure why we do this for toolbar/title and not for others like segment/segment-button?

```
  private getMode() {
    const toolbar = this.el.closest('ion-toolbar');
    return (toolbar && toolbar.mode) || this.mode;
  }

  hostData() {
    const mode = this.getMode();

    ...
  }
```

**Comment by brandyscarney** [comment]
This should be fixed with v5 - verify

**Comment by brandyscarney** [comment]
Confirmed fixed by `5.0.0-beta.5`, updated Codepen: https://codepen.io/brandyscarney/pen/bZZXrm
